furniro-front-end/src/app/header/Index.tsx

```diff
@@ -0,0 +1,3 @@
+import HeaderButtons from "./components/HeaderButtons";
+import Header from "./components/Header";
+export { HeaderButtons, Header };
```

Comment:
Try to separate the imports by adding comments describing what they are, like // Components, for components

Reply:
I added the comments, them being:

// Header components
import Header from "./components/Header";

and

// Buttons and Icons
import HeaderButtons from "./components/HeaderButtons";